features/image-transformations/image-enhancement-and-color-manipulation.md

```diff
@@ -133,3 +133,117 @@ URL - [https://ik.imagekit.io/demo/lipstick_2M.png](https://ik.imagekit.io/demo/
 {% endtab %}
 
 {% endtabs %}
+
+### Gradient - (e-gradient)
+
+This is used to add a gradient overlay over an input image.
+
+The gradient formed is a *linear gradient* (containing two colors) and it can be customised with the following parameters:
+
+| Parameter       | Description       | Value Type        | Range      | Default Value     |
```

Comment:
General point: In the `Default Value` column, can we use bulleted lists instead of just line breaks?

Reply:
I used bullets, but tbh line breaks look way better and cleaner in this particular case.

Reply:
resolved